lib/betamocks/middleware.rb

```diff
@@ -7,7 +7,7 @@
 require_relative 'response_cache'
 
 module Betamocks
-  class Middleware < Faraday::Response::Middleware
+  class Middleware < Faraday::Middleware
```

Comment:
noted in [upgrade docs](https://github.com/lostisland/faraday/blob/main/UPGRADING.md#others)